time/src/main/java/io/spine/time/string/TimeStringifiers.java

```diff
@@ -162,9 +186,116 @@ private static void registerAll() {
+     *
+     * <p>This stringifier is automatically registered in the
+     * {@link StringifierRegistry StringifierRegistry}.
+     *
+     * @see io.spine.time.YearMonths#toString(YearMonth) YearMonths.toString(YearMonth)
+     * @see io.spine.time.YearMonths#parse(String) YearMonths.parse(String)
+     */
+    public static Stringifier<YearMonth> forYearMonth() {
+        return YearMonthStringifier.getInstance();
+    }
+
+    /**
+     * Obtains default stringifier for {@code ZoneId}s.
+     *
+     * <p>This stringifier is automatically registered in the
+     * {@link StringifierRegistry StringifierRegistry}.
+     *
+     * @see io.spine.time.ZoneIds#toString(ZoneId) ZoneIds.toString(ZoneId)
+     * @see io.spine.time.ZoneIds#parse(String) ZoneIds.parse(String)
+     */
+    public static Stringifier<ZoneId> forZoneId() {
+        return ZoneIdStringifier.getInstance();
+    }
+
+    /**
+     * Obtains default stringifier for {@code ZonedDateTime}.
+     *
+     * <p>This stringifier is automatically registered in the
+     * {@link StringifierRegistry StringifierRegistry}.
+     *
+     * @see io.spine.time.ZonedDateTimes#toString(ZonedDateTime)
+     *      ZonedDateTimes.toString(ZonedDateTime)
+     * @see io.spine.time.ZonedDateTimes#parse(String)
+     *      ZonedDateTimes.parse(String)
+     */
+    public static Stringifier<ZonedDateTime> forZonedDateTime() {
+        return ZonedDateTimeStringifier.getInstance();
+    }
+
+    /**
+     * Obtains default stringifier for {@code ZoneOffset}s.
+     *
+     * <p>This stringifier is automatically registered in the
+     * {@link StringifierRegistry StringifierRegistry}.
+     *
+     * @see io.spine.time.ZoneOffsets#toString(ZoneOffset) ZoneOffsets.toString(ZoneOffset)
+     * @see io.spine.time.ZoneOffsets#parse(String) ZoneOffsets.parse(String)
+     */
+    public static Stringifier<ZoneOffset> forZoneOffset() {
+        return ZoneOffsetStringifier.getInstance();
+    }
+
+    /**
+     * Registers standard date-time stringifers at the {@link io.spine.string.StringifierRegistry
+     * StringiferRegistry}.
+     */
+    static class Registrar {
+
+        private final ImmutableList<Stringifier<?>> stringifiers;
+
+        private Registrar() {
+            stringifiers = ImmutableList.of(
+                    forDayOfWeek(),
+                    forDuration(),
+                    forLocalDate(),
+                    forLocalDateTime(),
+                    forLocalTime(),
+                    forMonth(),
+                    forOffsetDateTime(),
+                    forOffsetTime(),
+                    forTimestamp(),
+                    forYearMonth(),
+                    forZoneId(),
+                    forZonedDateTime(),
+                    forZoneOffset()
+            );
+        }
+
+        private void register() {
+            StringifierRegistry registry = StringifierRegistry.getInstance();
+            if (registry.get(Duration.class).isPresent()) {
```

Comment:
As long as you use `Duration.class` to understand if the following piece of code has been executed already, it also makes sense to check the same afterwards. If everything goes right, the duration is going to be present after the `.register(...)` loop ends.